test/pretty.test.js

```diff
@@ -66,6 +66,13 @@ test('throws when prettyPrint is true but pino-pretty module is not installed',
   require.cache[require.resolve('pino-pretty')].exports = prettyFactory
 })
 
+test('throws when prettyPrint has invalid options', async ({ throws, equal }) => {
+  throws(() => pino({ prettyPrint: { ignore: ['hostname'] } }))
+  try { pino({ prettyPrint: { ignore: ['hostname'] } }) } catch ({ message }) {
+    equal(message, 'opts.ignore.split is not a function')
+  }
```

Comment:
Yes, it's correct but poorly written. Try a more idiomatic way to write the same:

```suggestion
  throws(() => pino({ prettyPrint: { ignore: ['hostname'] } }), 'opts.ignore.split is not a function')
```
see https://node-tap.org/docs/api/asserts/#tthrowsfn-expectederror-message-extra

Reply:
I did rewrite all throws in tests that did follow bad pattern.